docker-compose.yml

```diff
@@ -18,36 +18,9 @@ mesosmaster:
   links:
     - zookeeper
 
-slave1:
+slave:
   build: .
   privileged: true
-  ports:
-    - "5051:5051"
```

Comment:
Is it still possible to follow the links in the Mesos UI to the slave sandbox without this mapping?